src/tequila/circuit/_gates_impl.py

```diff
@@ -6,7 +6,7 @@
 from tequila.objective.objective import Variable, FixedVariable, assign_variable
 from tequila.hamiltonian import PauliString, QubitHamiltonian, paulis
 from tequila.tools import list_assignment
-from numpy import pi, sqrt
+import numpy as np
```

Comment:
Just some small adjustment to not import `numpy` functionality twice. Let me know if this is ok for you, otherwise I'll just reverse it.